outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/domain/TargetFileAudit.java

```diff
@@ -0,0 +1,42 @@
+package org.motechproject.nms.outbounddialer.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+@Entity(tableName = "nms_target_file_audit")
+public class TargetFileAudit {
+    @Field
+    private String fileName;
+
+    @Field
+    private Integer recordCount;
+
+    @Field
+    private String checksum;
+
```

Comment:
Do you want to have a field in here to record the CDRs received at the end of the day as well? That way, we can easily correlate the data? probably feature creep for this checkin though